lib/sniffybara.rb

```diff
@@ -92,6 +96,12 @@ def find_accessibility_issues
     end
 
     def process_accessibility_issues
+      matched_excluded_path_index = Driver.path_exclusions.find_index do |path_pattern|
+        path_pattern =~ current_url
+      end
+
+      return if not matched_excluded_path_index.nil?
```

Comment:
I think you can probably shorten this using `Array.any?`

something like this:
```
return if Driver.path_exclusions.any? { |p| p =~ current_url }
```